.ci/update-docs.sh

```diff
@@ -8,6 +8,8 @@ cd docs
 
 # Update properties
 sed -i "s/CLIENT_VERSION=.*/CLIENT_VERSION=$1/g" gradle.properties
```

Comment:
Since `CLIENT_VERSION` is not used anywhere, its better to remove it.

Reply:
Please remove the following usage as well:

* `CLIENT_VERSION` in `gradle.properties`
* `client-version` in `build.gradle`.

Reply:
Done (in docs repository)